File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& num1, vector<int>& num2) {
        vector<int> num = num1;
        num.insert(num.end() , num2.begin() , num2.end());
        sort(num.begin() , num.end());
        int n = num.size();
        if(n % 2 == 0) 
            return (num[n/2] + num[n/2 - 1]) / 2.0 ;
        else
            return num[n/2];

    }
};

File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        lista = nums1 + nums2 
        lista.sort()
        n = len(lista)
        median = 0
        if (n % 2 == 0) :
            median = (lista[n//2] + lista[n//2 - 1]) / 2
        else:
            median = lista[n//2] 
        return median